quickstarts/ext-influxdb/config.yml

```
@@ -0,0 +1,79 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: ext-influxdb
+
+# Displayed in the UI (required)
+title: InfluxDB
+
+# Long-form description of the quickstart (required)
+description: |
+  The InfluxDB quickstart allows you to get visibility into the performance and available of your InfluxDB service and dependencies. Use this quickstart together with the mock up integrations.
+
+ # InfluxDB quickstart highlights
```

Looks like the yaml formatting is a little off, this should fix it!
```suggestion
  # InfluxDB quickstart highlights
```